### PR TITLE
Serialize JSON NaN as null

### PR DESCRIPTION
**Pull request by imbyungjun**
- Sometimes cpu usage percentage presents as NaN
- There's no NaN in JSON format, represent it as null

### CODE REVIEW

**Comment by donggyu04** [comment]
Can one of the admins verify this patch?

**Comment by imbyungjun** [comment]
https://github.com/naver/ngrinder/blob/d713428c8d084c231512f9d2484714279feff167/ngrinder-controller/src/main/java/org/ngrinder/agent/controller/AgentManagerApiController.java#L153

to serialize system data model, the number module have to be added to both objectMapper bean and JsonUtils

**Comment by junoyoon** [comment]
Very good!

File: ngrinder-frontend/src/js/components/perftest/mixin/FormatMixin.vue [hunk 1]
```diff
@@ -28,7 +28,10 @@
         }
 
         formatPercentage(format, value) {
-            value = value || 0;
+            if (!value) {
+                return 'N/A';
```
Comment (junoyoon) [inline ngrinder-frontend/src/js/components/perftest/mixin/FormatMixin.vue]: What about give space after A like ``` 'N/A '```

Comment (imbyungjun) [reply ngrinder-frontend/src/js/components/perftest/mixin/FormatMixin.vue]: I tried to add a space after it but, this value is wrapped by `<span/>` tag. So additional white space doesn't affect to html.
image

**Comment by junoyoon** [comment]
LGTM

**Pull request merged**